app/services/hyrax/listeners/metadata_index_listener.rb

```diff
@@ -35,9 +40,16 @@ def on_object_deleted(event)
 
       private
 
-      def log_non_resource(event)
+      def log_non_resource(event, idx = :object)
         Hyrax.logger.info('Skipping object reindex because the object ' \
-                          "#{event[:object]} was not a Valkyrie::Resource.")
+                          "#{event[idx]} was not a Valkyrie::Resource.")
+      end
+
+      def metadata_updated(event, idx)
```

Comment:
would it be cleaner to pass the resource through from the payload directly, instead of the `event` and `idx`?

so the signatures for the private methods become:

`def log_non_resource(obj)` and `def metadata_updated(resource)`